### ISSUE TITLE
Duplicated RegCM5 simulation by ICTP and CUNI

### ISSUE THREAD

**Issue by jesusff** [FPS-URB-RCC/STAGE-0_Analysis#20]
There are two RegCM5 simulations, one by ICTP and the other by CUNI. What is the difference between them, @anvipra?

The path for CUNI includes a folder CUNI_RegCM5, which should not have any underscore. It should be just RegCM5, according to the filename later on:

https://github.com/FPS-URB-RCC/STAGE-0_Analysis/blob/59bbc9f6d2683a99d345e14fc8bf74ea24e6a20d/list_server_files__pr_1hr.txt#L38

For ICTP, the model name is defined including the institution with a dash `ICTP-RegCM5`:

https://github.com/FPS-URB-RCC/STAGE-0_Analysis/blob/59bbc9f6d2683a99d345e14fc8bf74ea24e6a20d/list_server_files__pr_1hr.txt#L60

If this is intended to mean the "ICTP configuration of RegCM5" (as opposite to the CUNI configuration), then the filename should have ICTP twice, as
```
..._ICTP_ICTP-RegCM5_...
```
If these simulations were performed with the same configuration, but just on different machines/institutions, they should have the same `source_id = RegCM5` and different realization: `v1-fpsurbrcc-s0r1`vs `v1-fpsurbrcc-s0r2`

**Comment by jesusff** [comment]
Thank you for the prompt reply, Anahí. No, with so many differences, I would say to choose a different source_id (model name). Ideally, some name related to the main difference between them. Using a dash would be another option: CUNI-RegCM5, but this leads to redundant names such as `..._CUNI_CUNI-RegCM5_... nc` 

I'll try to have Johannes in, so you can take a consistent naming.

**Comment by hdeleeuw** [comment]
Sorry for the slight delay in the reply. I am in charge of the ICTP RegCM5 simulations and I agree that there are various different setting used and so the simulations are definitely not the same. This is the reason why I added ICTP-RegCM5, but I see that _ICTP_ICTP-RegCM5 isn't optimal use of the naming either (I will correct the ICTP file naming if we decide the name will be like this). It feels a bit redundant to have ICTP there twice and I'm happy to change it to something else if that would make more sense, but I have no good suggestions at the moment

**Comment by jesusff** [comment]
Thank you for your prompt reply. Could you summarize the differences between simulations here, so we can come up with a naming? From the details requested in the simulation list, they look the same, but there are no details about parameterizations other than the urban one there.
The naming can also be just as simple as RegCM5-A and RegCM5-B, or any other simple tag that can be later explained in the documentation.

**Comment by hdeleeuw** [comment]
I just had a look at the simulations and I think the main difference is the microphysics scheme used. There are also differences in the nudging at the boundaries of the domain, the projection and the start date of the simulation. Probably also the EUR-11 simulations used for nesting will have been run with similar differences, but I don't know this for sure. The CORDEX-CMIP6_source_id for the model is RegCM5-0, so maybe adding RegCM5-A or -B might lead to more confusion?

**Comment by anvipra** [comment]
You are right. I have just checked and there are also differences in the climatic ozone and aerosols database. In the case of CUNI, both options are set to 0, while ICTP has options 1 and 2, respectively. We could try to name them in a way similar to WRF simulations, something like RegCM5-CTRL for the ICTP simulations and RegCM5-MCH0 for CUNI, the last one meaning that the microphysics scheme and the chemistry databases used are different. What do you think?

**Comment by anvipra** [comment]
Our configuration is not exactly like RegCM5-0, so we will choose a different name (I'll discuss it with my colleague). We should also include a description in the metadata explaining the differences between the simulations. The questions are whether the ITCP simulation is RegCM5-0 (then, we will expand the differences between CUNI and ITCP simulations), or whether the ICTP simulation is a modification of RegCM5-0. In the latter case, should we explain the differences between ICTP and CUNI or between RegCM5-0 and CUNI?

**Comment by jesusff** [comment]
Regarding the FPS, what needs to be clear is the differences between the ICTP and CUNI runs, in order to interpret the different results. Where to document this, would be a topic to deal with in the FPS modellers team (@GLangendijk ). Currently, the info collected in the FPS metadata table does not allow to tell your runs apart.
